### DISCUSSION TITLE
What Cake version should current addin reference?

### DISCUSSION THREAD

**Discussion by MihaMarkic** [cake-build/cake#2978] [General]
What Cake version should current adding reference?
Is this the last stable (0.38.5 currently)?
Is there any page where it is written?

**Comment by devlead** [comment]
@MihaMarkic there's some good guidance is provided here currently 
https://github.com/cake-contrib/Home/blob/master/Guidelines.md

At this moment it's 0.33.0, but that'll be increased to 1.0 once that ships.

**Reply by pascalberger** [reply]
We also have an open issue to improve [documentation](https://cakebuild.net/docs/extending/addins/creating-addins ) with additional resources and best practices: https://github.com/cake-build/website/issues/1049

**Reply by pascalberger** [reply]
Best practices for writing Cake addins are now available at https://cakebuild.net/docs/extending/addins/best-practices.
